rotkehlchen/chain/evm/decoding/interfaces.py

```diff
@@ -170,9 +185,91 @@ def _decode_vote_cast(self, context: DecoderContext) -> DecodingOutput:
 
 
 class ReloadableDecoderMixin(metaclass=ABCMeta):
+    """Used by decoders of protocols that use data that can be reloaded from the db or from a
+    remote data source, to the decoder's memory."""
 
     @abstractmethod
     def reload_data(self) -> Optional[Mapping[ChecksumEvmAddress, tuple[Any, ...]]]:
         """Subclasses may implement this to be able to reload some of the decoder's properties
-        Returns only new mappings of addresses to decode functions
+        Returns only new mappings of addresses to decode functions"""
+
+
+class ReloadablePoolsAndGaugesDecoderMixin(ReloadableDecoderMixin):
```

Comment:
Shouldn't this also be an abstract class? It should never be instantiated on its own.